content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,337 @@
+lost reasons, as needed, where each lost reason occupies one rule row at a time.
+
+For more information, refer to the section below outlining how to :ref:`select appropriate lost
+reasons <email_marketing/select_lost_reasons>`.
+
+.. image:: lost_leads_email/reasons.png
+   :align: center
+   :alt: A list of filter rules that exclude all lost reasons other than the desired reason.
+
+Active
+------
+
+Finally, add a pair of :guilabel:`Active` filters to include both active and inactive leads for the
+campaign.
+
+.. important::
+   Adding both active **and** inactive lead records is necessary to capture the full scope of lost
+   leads in the database. Doing one without the other greatly impacts the number of targetable
+   records for the email campaign, and will not include a complete or accurate lost leads audience.
+
+Click the :guilabel:`(Add Branch)` icon to the right of the latest rule's row to add a pair of
+:guilabel:`any of` rules. In the top rule's first field, select the :guilabel:`Active` parameter
+from the drop-down menu. The rule will then automatically fill out to read: :guilabel:`Active` *is*
+`set`.
+
+For the first field of the bottom rule, once more, select :guilabel:`Active` from the drop-down
+menu and, this time, select :guilabel:`is not` from the operator drop-down menu, so the rule reads:
+:guilabel:`Active` *is not* `set`.
+
+.. image:: lost_leads_email/active.png
+   :align: center
+   :alt: A pair of Match Any Of filter rules that include both active and inactive leads.
+
+Add body content
+================
+
+Now, with the domain section of the email campaign complete, create the body content of the email
+using any of the premade stylized templates, or choose between the :guilabel:`Plain Text` or
+:guilabel:`Start From Scratch` options for more granular control. For more information, refer to the
+*Email Marketing* :ref:`documentation on how to create an email <email_marketing/create_email>`.
+
+.. tip::
+   To save the set of filters for later use, click :guilabel:`Save as Favorite Filter 💾`, enter a
+   name (such as `Lost Leads`), and click :guilabel:`Add`.
+
+   .. image:: lost_leads_email/favorite-filter.png
+      :align: center
+      :alt: The Save as Favorite Filter pop-up can save the lost leads criteria for later.
+
+Send or schedule
+================
+
+Once all the components of the email campaign are complete:
+
+- Click the purple :guilabel:`Send` button at the top-left of the form to immediately send the
+  email;
+
+  OR
+- Click gray :guilabel:`Schedule` button, located to the right of the :guilabel:`Send` button, to
+  send the email at a future date and time.
+
+.. tip::
+   Consider using A/B Testing to send an alternate version of the email to a percentage of the
+   target leads. This can help determine what subject lines and body content produce the best
+   click-through rates before sending a final version to the remaining leads.
```

Comment:
```suggestion
   Consider using *A/B Testing* to send an alternate version of the email to a percentage of the
   target leads. This can help determine what subject lines and body content produce the best
   click-through rates, before sending a final version to the remaining leads.
```
suggestion the use of italics here, as this reads as more of a brief definition/helpful tip